dev/instantsearch.css

```diff
@@ -1 +0,0 @@
-../dist/instantsearch.css
```

Comment:
It seems you removed the symlink here to dist/ (maybe by mistake/command). The goal is to load the builded CSS file in the dev env. Can you put it back?